### PR TITLE
[ISSUE #8318] fix findAllConfigInfoForDumpAll sql args error

### LINKED ISSUE #8318: nacos-config findAllConfigInfoForDumpAll method bug
**Describe the bug**
In there have sql parameters bug.

ExternalStoragePersistServiceImpl#findAllConfigInfoForDumpAll
image

EmbeddedStoragePersistServiceImpl#findAllConfigInfoForDumpAll
image

**Expected behavior**
return Page Object.

**Acutally behavior**
SQL Error

**Desktop (please complete the following information):**
 - Version [2.1.0]
 - Module [nacos-config]

**Resolvent**
Maybe

ExternalStoragePersistServiceImpl#findAllConfigInfoForDumpAll
new Object[] {(pageNo - 1) * pageSize, pageSize} instead of params.toArray()

EmbeddedStoragePersistServiceImpl#findAllConfigInfoForDumpAll
new Object[] {(pageNo - 1) * pageSize, pageSize} instead of EMPTY_ARRAY

Thanks.

### PR DESCRIPTION
**Pull request by onewe**
Close #8318

Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java [hunk 1]
```diff
@@ -1435,13 +1435,11 @@ public Page<ConfigInfoWrapper> findAllConfigInfoForDumpAll(final int pageNo, fin
...
         
-        List<String> params = new ArrayList<String>();
-        
         try {
-            return helper.fetchPageLimit(sqlCountRows, sqlFetchRows, params.toArray(), pageNo, pageSize,
+            return helper.fetchPageLimit(sqlCountRows, sqlFetchRows, new Object[] {(pageNo - 1) * pageSize, pageSize}, pageNo, pageSize,
                     CONFIG_INFO_WRAPPER_ROW_MAPPER);
         } catch (CannotGetJdbcConnectionException e) {
-            LogUtil.FATAL_LOG.error("[db-error] " + e.toString(), e);
+            LogUtil.FATAL_LOG.error("[db-error] " + e, e);
```
Comment (Oliverwqcwrw) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:1442]: IMO,Print the log once
`LogUtil.FATAL_LOG.error("[db-error] ", e);`

**Review by Oliverwqcwrw — CHANGES_REQUESTED** [review]

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java (hunk 1 above)
Comment (onewe) [reply config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:1442]: Ok, I will fix it

**Review by Oliverwqcwrw — APPROVED** [review]
LGTM

**Comment by KomachiSion** [comment]
Sorry, Can you rebase `develop` branch for this PR? we need run coverage in CI.

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**